### ISSUE TITLE
View All Request only shows current user requests

### ISSUE THREAD

**Issue by lindamrigo** [Fujitsu-America/Fujitsu-Repository#118]
Under PBS Troubleshooter, menu option View All Requests is only showing current user requests.  In 12.1.3, all requests regardless of user  were shown. It needs to show all requests.

**Comment by vinaytalasila** [comment]
Linda,

Is this happening to all the users who has the 'PBS Troubleshooter' responsibility in the Production instance?

Is this happening for your user in INT 12.2.10 instance so that we can troubleshoot the issue in this instance.

Thanks
Vinay

**Comment by lindamrigo** [comment]
It is happening in INT and every user that has PBS Troublshooter has the the same issue

**Comment by vinaytalasila** [comment]
**@ashoksnapfj:** Could you please check whether any roles are needed for the users for this to work? 

Thanks
Vinay

**Comment by vinaytalasila** [comment]
Update...

- We have created a custom Role and assigned that to the 'PBS Troubleshooter' responsibility in INT 12.2.10 instance. Tested and it works (Ashok and I can each others concurrent requests).
- Ashok will do more tests by Monday and prepare a configuration document so that Linda can review and test in INT 12.2.10 instance.
- If that works good then Linda can move the setups to Pre-Prod instance and Production instance.

Thanks
Vinay
